pkg/provider/common.go

```diff
@@ -31,3 +34,35 @@ func buildAPIReqAssignTags(tfRsrcTags []commonTerraform.Tag) []commonApi.Tag {
 	}
 	return tags
 }
+
+var resourceWal = schema.SingleNestedAttribute{
+	Description: "Write-Ahead Logs (WAL) Storage.",
+	Optional:    true,
+	Attributes: map[string]schema.Attribute{
```

Comment:
can also add more details to description like:

Use a separate storage volume for Write-Ahead Logs (Recommended for high write workloads)